app/controllers/sessions_controller.rb

```diff
@@ -14,8 +14,20 @@ def create
     end
   end
 
+  def update
+    current_user.github_token = gather_token
+    current_user.save(validate: false)
```

Comment:
nice, (validate: false), otherwise it will error because of password validation.